docs/optimization_guide/nncf/ptq/code/ptq_aa_openvino.py

```diff
@@ -58,6 +58,9 @@ def validate(model: openvino.CompiledModel,
 input_fp32 = ... # FP32 model input
 res = model_int8(input_fp32)
 
-# save the model
-ov.serialize(quantized_model, "quantized_model.xml")
+# save the model with compress_to_fp16=False to avoid an accuracy drop from compression
+# of unquantized weights to FP16. This is necessary because
+# nncf.quantize_with_accuracy_control(...) keeps the most impactful operations within
+# the model in the original precision to achieve the specified model accuracy
+ov.save_model(quantized_model, "quantized_model.xml", compress_to_fp16=False)
```

Comment:
compress_to_fp16=false is needed since model was optimized with fp32 precision during optimization?
Not sure if this is mandatory?

Reply:
It's not mandatory but recommended to guarantee the same accuracy as was achieved by AAQ applied on INT8+FP32 model. I would add a note explaining that if the original model is in FP32 then AAQ will revert some layers to FP32 back, but since save_model() saves it to FP16 then the accuracy can be impacted so compress_to_fp16=False is recommended in such case.

Reply:
@MaximProshin, @yury-gorbachev please take a look.